test/testrunner.py

```diff
@@ -402,6 +402,7 @@ def main():
                 raise Exception(f"EKS cluster {eks_cluster_name} is not in active state")
 
         # Execute dlc_tests pytest command
+        test_path = os.path.join(test_path, "test_efa.py")
```

Comment:
this will be reverted once PR is reviewed, this was added once all other tests were passing